integration/spark-common-test/src/test/resources/partition_data.csv

```diff
@@ -0,0 +1,27 @@
+id,vin,logdate,phonenumber,country,area,salary
```

Comment:
can you try to reuse the current csv files or generate data. 
Don't suggest adding so many csv file to repo.